rep/db/db.py

```diff
@@ -44,15 +46,66 @@ def create_db(self):
                 unixTime INTEGER NOT NULL,
                 billNumber VARCHAR NOT NULL,
                 voteName VARCHAR NOT NULL,
-                repName VARCHAR NOT NULL,
+                repId INTEGER NOT NULL,
                 repVote VARCHAR NOT NULL,
                 rawVoteObjectId INTEGER NOT NULL,
                 FOREIGN KEY (rawVoteObjectId) REFERENCES raw_vote_object(id),
-                PRIMARY KEY (rawVoteObjectId, repName)
-            )
+                PRIMARY KEY (rawVoteObjectId, repId)
+            );
+
+        CREATE TABLE IF NOT EXISTS representative_info
+            (
+                id INTEGER PRIMARY KEY,
+                dist VARCHAR NULL,
+                officeCode VARCHAR NULL,
+                districtNumber VARCHAR NULL,
+                designatorCode VARCHAR NULL,
+                firstName VARCHAR NULL,
+                middleInitial VARCHAR NULL,
+                lastName VARCHAR NULL,
+                suffix VARCHAR NULL,
+                commonlyUsedName VARCHAR NULL,
+                homeStreetAddress VARCHAR NULL,
+                homeCity VARCHAR NULL,
+                homeState VARCHAR NULL,
+                homeZipCode VARCHAR NULL,
+                homePhone VARCHAR NULL,
+                capitolStreetAddress VARCHAR NULL,
+                capitolCity VARCHAR NULL,
+                capitolPhone VARCHAR NULL,
+                room VARCHAR NULL,
+                roomNumber VARCHAR NULL,
+                committeesChaired VARCHAR NULL,
+                committeesViceChaired VARCHAR NULL,
+                rankingMember VARCHAR NULL,
+                committeeMember1 VARCHAR NULL,
+                senatorRepresentative VARCHAR NULL,
+                party VARCHAR NULL,
+                title VARCHAR NULL,
+                gender VARCHAR NULL,
+                businessPhone VARCHAR NULL,
+                email VARCHAR NULL,
+                fax VARCHAR NULL,
+                prison VARCHAR NULL,
+                url VARCHAR NULL,
+                committeeCodes VARCHAR NULL
+            );
         ''')
 
         conn.commit()
+
+        filename = 'database/data/LegislatorDatabase_2020.csv'
+        with open(filename) as csv_file:
+            reader = csv.reader(csv_file)
+            next(reader)
+            for field in reader:
+                representative_info_dao = RepresentativeInfoDao()
+                # I can't get the auto-incrementing ID to work unless I give
```

Comment:
that's confusing, because you're explicitly passing `null` in the insert statement https://github.com/LoveShack-Inc/represent/pull/23/files#diff-9d3653787a8570bb14f8a4c5e7240c4a4eb62c7ee2a2e70910471b69dfc862fcR113

Reply:
I am indeed confused, I don't know why it works that way, but I couldn't get anything else to work. I don't know much about databases, so I'm sure there's a way to insert a field into a database when you don't already have a value for the auto-incrementing ID, but whatever SQL statements I tried didn't work. I think I tried to use ProcessedVoteResultDao as a template but I wasn't successful, but I can try again